jetty-util/src/test/java/org/eclipse/jetty/util/security/PasswordTest.java

```diff
@@ -44,4 +56,35 @@ public void testObfuscateUnicode()
         String obfuscate = Password.obfuscate(password);
         assertEquals(password, Password.deobfuscate(obfuscate));
     }
+
+    @Test
+    public void testCommandLineUsage() throws IOException, InterruptedException
+    {
+        ProcessBuilder passwordBuilder = new ProcessBuilder()
+            .directory(MavenTestingUtils.getTargetDir())
+            .command("java",
+                "-cp", MavenTestingUtils.getTargetPath("classes").toString(),
+                Password.class.getName(),
+                "user", "password")
+            .redirectErrorStream(true);
+
+        Process passwordProcess = passwordBuilder.start();
+        try (InputStreamReader inputStreamReader = new InputStreamReader(passwordProcess.getInputStream());
+             BufferedReader reader = new BufferedReader(inputStreamReader))
```

Comment:
Inline `inputStreamReader`, it does not need to be listed as a resource, since it's not used within the block.